src/Streamly/Internal/System/Command.hs

```diff
@@ -64,15 +64,12 @@ import Data.Word (Word8)
 import Streamly.Data.Array (Array)
 import Streamly.Data.Fold (Fold)
 import Streamly.Data.Stream (Stream)
-import Streamly.Data.Stream.Concurrent (MonadAsync)
+import Streamly.Data.Stream.Prelude (MonadAsync)
```

Comment:
If we are importing Streamly.Data.Stream.Prelude then the import of Streamly.Data.Stream is redundant because it is included in Prelude.